app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -358,6 +358,7 @@ export default {
         },
       ],
       selectedTabIndex: 0,
+      test: 'testval',
```

Comment:
Did you forget to remove this? Should it be blank text?

Reply:
Yes. It is not used anywhere

Reply:
Pushed by error, have updated it now https://github.com/chatwoot/chatwoot/pull/2952/commits/7ef92e90550e4922105228aa9a4d9ec5ed158a77